README.md

```diff
@@ -107,6 +106,8 @@ After you're finished please remove all the comments and instructions!
 
 > View Video Demo [here](https://www.loom.com/share/0ff174560ebd4edfa4ee68f389f36836)
 
+- View Video Demo [here] video1584519968.mp4
+
```

Comment:
- [ ] You have provided a link to your new video presentation but I am not able to figure out how can i use this link to view your presentation :thinking: Kindly add a proper link to your video so next reviewer will be able to see your video and also to make your readme file descriptive and professional :+1: